Apps/Admin/Server/appsettings.json

```diff
@@ -60,8 +60,8 @@
     "ContentSecurityPolicy": {
         "Base": "'self'",
         "DefaultSource": "'self'",
-        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha512-uKQ39gEGiyUJl4AI6L+ekBdGKpGw4xJ55+xyJG7YFlJokPNYegn9KwQ3P8A7aFQAUtUsAQHep+d/lrGqrbPIDQ==' 'sha256-Tui7QoFlnLXkJCSl1/JvEZdIXTmBttnWNxzJpXomQjg=' 'sha256-vY0Cwh5hLv8sUQ61m3KQgfrqeAmcY7t6wJ2FAdYPNyU='",
-        "ConnectSource": "'self' file: data: blob: filesystem: https://spt.apps.gov.bc.ca/com.snowplowanalytics.snowplow/tp2 https://*.loginproxy.gov.bc.ca/ https://loginproxy.gov.bc.ca/",
+        "ScriptSource": "'self' 'wasm-eval' 'unsafe-eval' 'sha256-nYaPGPw9zQuQHVl09VRaW597Ut8F9Lf/VDNxGSQu3Sc=' https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/js/all.min.js",
```

Comment:
Similar comment as on last ScriptSource change: we should keep 'sha256-Tui7...' and 'sha256-vY0C...' as they're for Swagger.

https://github.com/bcgov/healthgateway/pull/5548#pullrequestreview-1621049481

Reply:
Nice catch, I will add those back.